doc/docs/Python_Tutorials/Eigenmode_Source.md

```diff
@@ -106,7 +108,13 @@ There are numerical dispersion artifacts due to the FDTD spatial and temporal di
+We can demonstrate that the total power in a waveguide with *arbitrary* orientation — computed using two equivalent methods via `get_fluxes` and [mode decomposition](../Mode_Decomposition.md) — can be computed by a single flux plane oriented along the $y$ direction: thanks to [Poynting's theorem](https://en.wikipedia.org/wiki/Poynting%27s_theorem), the flux through any plane crossing a lossless waveguide is the same, regardless of whether the plane is oriented perpendicular to the waveguide. Furthermore, the eigenmode source is normalized in such a way as to produce the same power regardless of the waveguide orientation — in consequence, the flux values for mode **A** of the single-mode case for rotation angles of 0°, 20°, and 40° are 1111.280794, 1109.565028, and 1108.759159, within 0.2% (discretization error) of one another. Note that the Poynting flux could have been normalized to unity by setting the `EigenModeSource`/`Source` object parameter `amplitude=1/src.fourier_transform(fsrc)` where `fsrc=0.15` and `src=mp.GaussianSource(fsrc,fwidth=0.2*fsrc)`.
+
+Finally, we demonstrate that as long as the line source intersects the waveguide *and* `eig_kpoint` is not nearly parallel to the direction of the line source, the mode can be properly launched. As shown in the field profiles below for the single-mode waveguide, there does not seem to be any noticeable distortion in the launched mode as the waveguide approaches glancing incidence to the source plane up to 80°, where the total power in the forward-propagating mode is 97%. Note that the line source spans the entire length of the cell extending into the PML region (not shown). In this example, the maximum rotation angle is ~84°, where the total power drops to 59% and the backward-propagating fields are clearly visible. Due to discretization errors, increasing the size of the cell will *not* improve results beyond 84°, unless you also increase resolution.  As the waveguide mode approaches glancing incidence, sensitivity to discretization errors increases (because the mode varies rapidly with frequency on a glancing-angle cross-section).  For waveguide angles much beyond 45° you probably want to simply change the orientation of the line source by 90°.
+
```

Comment:
Would be good to actually try increasing the cell size for the 83° and 84° cases, to see if the resolution is limiting.

Reply:
Since you tried it and increasing the cell size helps, it would be good to show the larger cell size (for which there are still resolution effects)

Reply:
Done.